src/collections/_documentation/workflow/discover2/query-builder.md

```diff
@@ -0,0 +1,333 @@
+
+### Renaming a Saved Query
+
+1. Click on the Saved Query name with the pencil icon 
+2. Enter a new name
+3. Click enter or click outside to confirm changes
+4. The name is automatically saved
+
+### Deleting a Saved Query
+
+{% capture __alert_content -%}
+Deleting a Saved Query is irreversible.
+{%- endcapture -%}
+{%- include components/alert.html
+    title="Warning"
+    content=__alert_content
+    level="warning"
+%}
+
+From an existing Saved Query, click on the garbage bin icon on the top right-hand corner. After deleting a Saved Query, you'll be redirected back to the landing page.
+
+## Search Conditions
+
+You can use conditions to filter events from results by fields or tags.
+
+### Available Fields
+
+Events have a number of built in fields as well as custom tags.
+
+#### Standard fields
+
+<table class="table">
+{%- for category in page.standard_fields -%}
+  <tr>
+    <td>
+      <strong>{{ category[0] }}</strong>
+    </td>
+    <td class="content-flush-bottom">
+      {%- assign items = category[1]-%}
+      {%- include api/params.html params=items -%}
+    </td>
+  </tr>
+{%- endfor -%}
+</table>
+
+### Syntax
+
+The Query Builder syntax is identical to [Sentry's Search syntax]({%- link _documentation/workflow/search.md -%}). To add a search condition, click on the search input box above the graph, add a search condition, and press "Enter."
+
+### Searching on aggregated fields/tags
+
+You can add search conditions on aggregated fields/tags (for example, `count(id)`) in one of the following ways:
+
+- Exact match: `count(id):99`
+- Upper bounds: `count(id):<99` or `count(id):<=99`
+- Lower bounds: `count(id):>99` or `count(id):>=99`
+- Combination of upper and lower bounds: `count(id):>10 count(id):<20`
+
+## Results Table
+
+The results of the table are generated based on the following components:
+
+- Selected columns
+    - Columns may either be a known field, or a tag
+    - Columns may have an aggregation function applied to them
+    - Sort order depends on the column selected
+- Search conditions
+- Global selection header (projects, environment, dates)
+
+### Graph
+
+Each Discover query has a graph that displays a visual summary of the data presented in the results table. Releases are overlaid, clickable, and the data corresponding to the previous period is also shown. Additionally, you can zoom in on specific slices to drill into a spike or problem area.  
+
+By default, the y-axis shows the count of events over time. However, you can display  `count` or `count_unique` on almost any column if you add it as a column in the query builder. Note that the y-axis will reset to another available option if you remove the column from the table.
+
+### Aggregation functions
+
+- `avg`
+- `count`
+- `count_unique`
+- `max`
+- `min`
+- `sum`
+
```

Comment:
I think it would be extremely useful to maybe link to another page that discusses building a new query from scratch and the process for it. I believe it would drive home a lot of the features documented here with a concrete example.

Most of our customers will be familiar with SQL as a query language and will want to know how to map those concepts to Discover. For example, this documentation doesn't really talk about the concept of grouping in any formal sense. So having an exemplar query being built and talking through these concepts would really drive home the mental model and can provide links to these sections as needed. I personally had a tough time with this, so I would love to help here ;-)

Reply:
+1 . Let's not hold up the launch for this but let's build a page of Query samples -- especially based on real ones that Neil, Will & Bowen etc have been working on with customers.   I'll even write up my Healthy Sessions hack if we have to

Reply:
Zac and @idosun will discuss creating this tutorial for the Guides section of Docs.